duckduckgo/chrome-zeroclickinfo
# Add DDG styles

**Reviewer:** @jdorweiler



## Description:

Following up from #61, add DDG colors and litestrap framework, for consistency with other project and for ease of UX development.

## Steps to test this PR:
We're not adding new features, so we want to verify the colors in the popup are the right ones.
1. Load any webpage
2. Open the popup
3. Hover on the search button: it should be green
4. Links are light blue
5. Reload button is blue
6. Generally everything looks just as before

Notes: thanks to using the duckduckgo-colors module, now we're going to display the right shade of green for the search button. See [this commit](https://github.com/duckduckgo/duckduckgo-colors/commit/b85745fbef95fbd122be3a030dd89f9361756a72) in the duckduckgo-colors repo.


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
